desktop/src/gui/controller.rs

```diff
@@ -190,14 +192,28 @@ impl GuiController {
             }
         }
 
-        if let winit::event::WindowEvent::ThemeChanged(theme) = &event {
+        if let WindowEvent::ThemeChanged(theme) = &event {
             let visuals = match theme {
                 Theme::Dark => egui::Visuals::dark(),
                 Theme::Light => egui::Visuals::light(),
             };
             self.egui_winit.egui_ctx().set_visuals(visuals);
         }
 
+        if matches!(
+            &event,
+            WindowEvent::KeyboardInput {
+                event: winit::event::KeyEvent {
+                    logical_key: Key::Named(NamedKey::Tab),
+                    ..
+                },
+                ..
+            }
+        ) {
+            // Prevent egui from consuming the Tab key.
```

Comment:
On #13426, it was stated that
> The TAB key is used to access the inventory in the game, but in the Ruffle desktop player, TAB moves through Ruffle's own menus and doesn't forward the key to the game. Fortunately, you can also open the inventory by clicking the backpack with the mouse...

Does this change fix that?

Reply:
That is true, it should be fixed now.